Assignment6/CParkingLot.h

```diff
@@ -0,0 +1,28 @@
+#ifndef REARRANGING_CARS_CPARKINGLOT_H
+#define REARRANGING_CARS_CPARKINGLOT_H
+
+#include <iostream>
+#include <map>
+#include <vector>
+
+class CParkingLot {
+private:
+    int NumberOfPlaces_;
```

Comment:
I think it's better to split with blank lines the main fields of the class and fields which are needed for internal methods and algorithms.